packages/rirc/makefile.patch

```diff
@@ -0,0 +1,10 @@
+--- rirc/Makefile	2022-09-12 23:12:14.492293684 +0530
++++ rirc.mod/Makefile	2022-09-12 18:39:34.298457909 +0530
+@@ -1,6 +1,6 @@
+ VERSION := 0.1.6
+ 
+-PREFIX   ?= /usr/local
++PREFIX   ?=  @TERMUX_PREFIX@/local
```

Comment:
This patch shouldn't be necessary, it uses `?=` and so should be overridden by the value we set

Reply:
Thanks, I don't know this